src/main/java6/net/finmath/marketdata/model/curves/AbstractForwardCurve.java

```diff
@@ -145,4 +119,24 @@ public double getPaymentOffset(double fixingTime) {
 
 		return values;
 	}
+	
+	@Override
+	public double getForward(double fixingTime)
+	{
+		return this.getForward(null, fixingTime, getPaymentOffset(fixingTime));
+	}
+	
+	@Override
+	public String getDiscountCurveName() {
+		return null;
```

Comment:
Why? Suddenly returning null where we previously returned the name of the associated discount curves may break a whole lot of code. Why do forward curves suddenly no longer have the possibility of carrying the associated discount curve.